### DISCUSSION TITLE
Cannot find module '@radix-ui/react-tooltip' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by malavikakoppula** [radix-ui/primitives#827] [Help]
Cannot find module '@radix-ui/react-tooltip' or its corresponding type declarations. 
Cannot find module '@radix-ui/react-slot' or its corresponding type declarations.  TS2307

import React, { ReactElement } from "react";
import { Root, Trigger, Content } from "@radix-ui/react-tooltip";
                                                                  ^
import { Slot } from "@radix-ui/react-slot";
                                   ^
function Tooltip(): ReactElement {

**Comment by jjenzz** [comment]
Would you mind providing some more information on your setup here? The type declarations are fine for me in a CodeSandbox at the moment https://codesandbox.io/s/pensive-fermi-3iwi7

**Reply by liamtarpey** [reply]
Same issue for me, using v0.1.6 with Typescript. In your sandbox it's an older version

**Reply by benoitgrelard** [reply]
Do you have a link to a broken sandbox?
Quite often it's just caching on codesandbox that's broken…

**Reply by jjenzz** [reply]
Additionally, I've updated the sandbox above to the latest and all appears to be fine still. 

@liamtarpey It'd be helpful to understand your setup more, even more so if you could create a sandbox that replicates it.

**Comment by joelbqz** [comment]
If someone experience this issue and the package seems to be installed in `package.json` try re-installing:

`yarn remove @radix-ui/react-select`
`yarn add @radix-ui/react-select`

That fixed the issue for me

**Reply by devxdemon** [reply]
Yes that worked for me too, thanks @joelbqz

**Reply by SaiRev0** [reply]
Didn't work for me using pnpm

**Reply by devxdemon** [reply]
@SaiRev0 just uninstall and install again using pnpm

**Reply by HEVpro** [reply]
Using pnpm it doesn't work for me neither

**Reply by SaiRev0** [reply]
@rahu1gg  So I don't know this, but using yarn solved the issue, but using npm or pnpm does not.

**Reply by bettjesse** [reply]
worked for me using npm, initial error was
Cannot find module '@radix-ui/react-dialog' or its corresponding type declarations.ts(2307) No quick fixes available
I deleted the package  and installed it again

**Reply by MikeCharpin** [reply]
`npm uninstall @radix-ui/react-slider`
`npm install @radix-ui/react-slider`

Fixed the problem for me.

**Reply by musiur** [reply]
Please after installation if you found this kind of error then try to restart VSCODE at least for once.

**Reply by vaughanlove** [reply]
or hard reset TS server. ctrl + shift + p, search and run "TypeScript: Restart TS Server"

**Comment by Risbel** [comment]
> Didn't work for me using pnpm
you have to run `pnpm uninstall` instead `remove`, (`pnpm uninstall @radix-ui/react-tabs` and next again `pnpm add @radix-ui/react-tabs`) 
that works for me

**Comment by Hasannm17** [comment]
npm uninstall @radix-ui/react-tooltip
then run ,  
npm install @radix-ui/react-tooltip

**Comment by nurikaraca** [comment]
My solution was using yarn.

**Reply by DanielCruz12** [reply]
> My solution was using yarn.

I removed --turbopack from dev in package.json, leaving only next dev. It works fine with pnpm without needing to switch to yarn or make any other changes

**Comment by pranay-dutta** [comment]
I debugged 1hr to find the bug and had 20 build fails on vercel.
And then used npm over pnpm and it magically fixed the issue.
